boggle.js

```diff
@@ -33,12 +33,13 @@ function randomInteger(min, max) {
 function shake(boggleBoard) {
   for (let row of boggleBoard)
   {
```

Comment:
Put these on the same line as the `for`. I'm surprised ESlint doesn't insist on it, but this style isn't really used in JS land.